group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -15086,6 +15087,63 @@ public void testShareGroupDescribeOnConsumerGroup() {
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void testReplayConsumerGroupRegularExpression() {
+        GroupMetadataManagerTestContext context = new GroupMetadataManagerTestContext.Builder()
+            .build();
+
+        ResolvedRegularExpression regex = new ResolvedRegularExpression(
```

Comment:
nit: resolvedRegularExpression there are 2 more instances in other tests

Reply:
You're picky :).